docs/pages/modules/module-api.mdx

```diff
@@ -354,6 +354,115 @@ Defines the activity lifecycle listener that is called when the activity owning
 
 </APIBox>
 
+<APIBox header="GroupView" platforms={["android"]}>
+
+Enables the view to be used as a view group. Definition components that are accepted as part of the group view definition: [`AddChildView`](#addchildview), [`GetChildCount`](#getchildcount), [`GetChildViewAt`](#getchildviewat), [`RemoveChildView`](#removechildview), [`RemoveChildViewAt`](#removechildviewat).
+
+#### Arguments
+
+- **viewType** — The class of the native view. Note: the provided class must inherit from the Android `ViewGroup`.
```

Comment:
```suggestion
- **viewType** — The class of the native view. Note that the provided class must inherit from the Android `ViewGroup`.
```

If a "Note" is not being called out, I'd suggest using it as a phrase in the sentence.